source/_components/image_processing.tensorflow.markdown

```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: "TensorFlow"
+description: "Detect and recognize objects with TensorFlow."
+date: 2018-10-24 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tensorflow.png
+ha_category: Image Processing
+ha_iot_class: "Local Polling"
+ha_release: 0.82
+---
+
+The `tensorflow` image processing platform allows you to detect and recognize objects in a camera image using [TensorFlow](https://www.tensorflow.org/). The state of the entity is the number of objects detected, and recognized objects are listed in the `summary` attribute along with quantity. The `matches` attribute provides the confidence `score` for recognition and the bounding `box` of the object for each detection category.
+
+<p class='note warning'>
+  To work with Hassbian after following the setup and installing these packages:
+  `$ sudo apt-get install libatlas-base-dev libopenjp2-7 libtiff5`
+</p>
+
+## {% linkable_title Setup %}
+
+You need install `tensorflow` Python packages with: `$ pip3 install tensorflow`. The wheel are not available for all platforms, it could be that you need compile it from source. Hass.io have this already preinstalled.
```

Comment:
Since we are now providing TF install instructions, would it make sense to also include the command for Nvidia one as well. 

You need install `tensorflow` Python packages with: `pip3 install tensorflow` or `pip3 install tensorflow-gpu` (for those who use a Nvidia GPU).

Reply:
Resolved with a link to the official install page on tensorflow.org